srcpkgs/python3-txtorcon/template

```diff
@@ -1,17 +1,19 @@
 # Template file for 'python3-txtorcon'
 pkgname=python3-txtorcon
-version=20.0.0
-revision=2
+version=21.1.0
+revision=1
 wrksrc="txtorcon-${version}"
 build_style=python3-module
 hostmakedepends="python3 python3-setuptools"
 depends="python3-automat python3-Twisted"
+checkdepends="lsof python3-mock python3-Twisted tox"
 short_desc="Twisted-based asynchronous Tor control protocol implementation"
 maintainer="travankor <travankor@tuta.io>"
 license="MIT"
 homepage="https://txtorcon.readthedocs.io/en/latest/"
 distfiles="https://github.com/meejah/txtorcon/archive/v${version}.tar.gz"
-checksum=8b5f5587ab03bb965c4d162b8ef86c6853c06041b200f2f8a2272caa4bd7cac4
+checksum=61ef40e110a2b52255a6033293d02647dedb63ed9311ce572571c777b2302750
+make_check=ci-skip
```

Comment:
could you explain why in a comment?

Reply:
is [this](https://pipelines.actions.githubusercontent.com/GbMGR0ISmtcNwrR1js8irbYChG8pi1T6JBrmuKRQmPX1bdcFkV/_apis/pipelines/1/runs/41520/signedlogcontent/5?urlExpires=2021-08-22T10%3A33%3A49.2317977Z&urlSigningMethod=HMACV1&urlSignature=UzMgbluztX2bBZ5XA6rIzqF8ThSny2UveLFllmxo%2FZ0%3D) log as a comment alright?

Reply:
I am not really sure why the failure is happening, maybe something with GH limiting open ports? Let's leave it without a comment then.